app/lib/common/providers/room_providers.dart

```diff
@@ -259,7 +259,7 @@ final _roomAvatarProvider = FutureProvider.autoDispose
 
 /// Provide the AvatarInfo for each room. Update internally accordingly
 final roomAvatarInfoProvider =
-    StateProvider.autoDispose.family<AvatarInfo, String>((ref, roomId) {
+    StateProvider.family<AvatarInfo, String>((ref, roomId) {
```

Comment:
side note that this doesn't need to be a `State` Provider either:
```suggestion
    Provider.family<AvatarInfo, String>((ref, roomId) {
```